index.html

```diff
@@ -0,0 +1,521 @@
+                  enjoyable. The knowledgeable and supportive community is an
+                  added bonus. Whether you're a seasoned athlete or just
+                  starting, this platform provides everything you need for a
+                  successful fitness journey. I highly recommend it to anyone
+                  committed to achieving their health and fitness goals!"
+                </p>
+              </blockquote>
+              <div class="flex mt-6 space-x-4 text-sm">
+                <span>Judith Black</span>
+                <span>CEO of Tuple</span>
+              </div>
+            </figure>
+          </div>
+        </div>
+      </section>
+      <!-- CTA Banner section-->
+      <section class="max-w-screen-2xl mx-auto mt-32 relative">
+        <a href="#">
+          <img
+            src="/assets/images/tiffany-nutt-0ClfreiNppM-unsplash 1.png"
+            alt="Orange Bicycle"
+            class="inset-0 w-full h-full object-cover"
+          />
+          <div class="cta-textbox absolute">
+            <h2 class="text-5xl text-travel-orange font-bold pb-8 max-w-lg">
+              Our biggest sale is now live!
+            </h2>
+            <div class="mt-16">
+              <a href="#">
+                <button
+                  class="bg-travel-orange text-lg font-semibold text-white px-4 py-2 rounded-xl"
+                >
+                  Shop Now
+                </button>
+              </a>
+            </div>
+          </div>
+        </a>
+      </section>
+      <!-- Newsletter -->
+      <section class="mt-16 px-12 py-12">
+        <div
+          class="max-w-7xl mx-auto grid grid-cols-1 gap-10 border border-travel-orange py-12 px-12 shadow-2xl rounded-lg lg:grid-cols-12"
+        >
+          <div class="max-w-xl text-3xl font-bold md:text-4xl lg:col-span-7">
+            <h2 class="inline">Want product news and updates?</h2>
+            <p class="inline">Sign up for our newsletter.</p>
+          </div>
+          <form class="max-w-md w-full self-center lg:col-span-5">
+            <div class="flex gap-x-4">
+              <label for="email-address" class="sr-only">Email address</label>
+              <input
+                id="email-address"
+                name="email"
+                type="email"
+                autocomplete="email"
+                required
+                class="flex-auto rounded-md px-3 py-2 text-black shadow-sm ring-1 ring-inset ring-travel-orange focus:ring-2 focus:ring-inset focus:ring-travel-orange md:text-sm border-travel-orange"
+                placeholder="Enter your email"
+              />
+              <button
+                type="submit"
+                class="flex-none rounded-md bg-travel-orange px-3.5 py-2.5 text-white text-sm font-semibold shadow-sm hover:bg-orange-600 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-yellow-500"
+              >
+                Subscribe
+              </button>
```

Comment:
In future probably want this to redirect to a success page - potentially add a 'Registration Successful' popup

Reply:
Yeah we can add this as tech debt in a new branch if you want to action this?